js/user.js

```diff
@@ -19,7 +19,7 @@ const setUserGreeting = (username, firstName, userProfilePicture) => {
     const userImgURL = userProfilePicture;
 
     userImgEl.forEach((element) => {
-      element.src = userImgURL;
+      element.src = userImgURL || '/img/profile.png';
```

Comment:
Is the purpose of this statement to ensure if the userImgURL is undefined or null then use /img/profile.png?

Do you think this would be helpful
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
I did think of that but Ankush mentioned the specified code

Reply:
Ok, do you know any specific reason for going ahead with this?

Reply:
Good discussion, We came to conclusion to go ahead with || operator here as we might as well receive `''`